news/4.37/platform_isv.md

```diff
@@ -16,6 +16,11 @@ See [Support Launching with a Terminal Console](platform.md#support-launching-wi
 
 ### Perspective Switcher Menu Can Now Be Hidden Programmatically
 
+<details>
+<summary>Contributors</summary>
+
+- [Lars Vogel](https://github.com/vogella)
+</details>
```

Comment:
I think a blank after details is maybe important. On phone so can’t check.

Reply:
I see both in https://raw.githubusercontent.com/eclipse-platform/www.eclipse.org-eclipse/refs/heads/master/news/4.37/platform.md the first from Christoph did not have an empty line, the rest have it. To be on the safe side, I add a blank line.

Reply:
Yes, it appears not to matter with respect to rendering, but it's kind of nicer when looking at the raw markup because there will be a blank line at that demarcation when rendered.